docs/blog/posts/v1.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2025-03-13
+---
+
+# Mesop v1
+
+After developing Mesop for nearly a year and a half and publicly launching it on [Hacker News](https://news.ycombinator.com/item?id=40567327) 9 months ago, we are excited to release Mesop v1.
+
+It's been an amazing journey working on Mesop, especially with [richard-to@](http://github.com/richard-to) who's been my 20% collaborator at Google. Despite being an extremely small core team (working on this as a 20% project!), we've been able to foster a thriving community both within Google where hundreds of Googlers have used it to build internal apps and outside of Google where it's been downloaded [more than 180k times](https://pepy.tech/projects/mesop) on PyPI.
+
+## What v1 means
+
+Mesop v1 signifies our commitment to prioritizing stability and avoiding backwards-incompatible changes unless absolutely necessary—such as important security fixes or critical bug fixes.
+
+Since we publicly launched 9 months ago, we have taken a conservative approach to making API changes, avoiding backwards-incompatible changes as much as possible. This has been key to Mesop's successful adoption both within Google and externally. As a small team, this approach was essential; we simply didn’t have the bandwidth to migrate the various Google teams using Mesop. In addition, Google's [reputatation for churn](https://abseil.io/resources/swe-book/html/ch15.html#:~:text=There%E2%80%99s%20an%20old,and%20fast%2Dpaced.) has made us careful in designing our APIs.
```

Comment:
"bandwidth to migrate the various Google teams using Mesop" -> I found this phrase slightly awkward.